### ISSUE TITLE
friends-swarm -> friends-db + friends-swarm

### ISSUE THREAD

**Issue by hackergrrl** [moose-team/friends-swarm#7]
It'd be helpful to see the core p2p database part of this module broken out into its own module, with `friends-swarm` layered on top. `friends-db` then could just expose db ops and generic replication streams that would allow for additional, non-webrtc transports too.

**Comment by bcomnes** [comment]
Agreed.  Also https://github.com/substack/chatwizard does something similar.
